source/threads.tex

```diff
@@ -1207,6 +1207,9 @@
   // hash support
   template<class T> struct hash;
   template<> struct hash<thread::id>;
+
+  template<class charT>
+  struct formatter<thread::id, charT>;
```

Comment:
Put on a single line, similar to std::hash specialization above?

Do we need a declaration of the primary class template here, similar to std::hash?

Reply:
That isn't done anywhere yet.